kernel/search/plugins/ezsearchengine/ezsearchengine.php

```diff
@@ -185,16 +184,17 @@ function buildWordIDArray( $indexArrayOnlyWords )
                 // Build a has of the existing words
                 $wordResCount = count( $wordRes );
                 $existingWordArray = array();
+                $wordIDArrayChuck = array();
```

Comment:
could we just move init of `$wordIDArray` here to avoid needing to change lines below?